Roadmap.md

```diff
@@ -35,3 +35,11 @@ Which of these should block releases?
 * TAP 3 & 4, work on implementations
 * Implement TAP 13 in go-tuf
 * All implementations should catch up to 1.x.y by $MONTH and $DATE?
+
+## Implementation roadmaps
+
+The following are links to roadmaps for various TUF implementations. These roadmaps are maintained by the projects. If you have a TUF implementation, feel free to open a pr to add it here.
+
+* **python-tuf**: 
+* **go-tuf**: https://github.com/orgs/theupdateframework/projects/3
```

Comment:
I know we don't have a dedicated roadmap yet, but it's worth noting - should we assume that a roadmap (something more high-level) would be the same as a project plan (i.e. like what we have so far in go-tuf which is a bit more detailed)?

Reply:
I would like go-tuf to have a higher-level roadmap. I think that would serve a different-but-important use compared with the project board.

Reply:
+1 for that 👍 I imagine the roadmap as a high-level plan that is being revisited every quarter or so.